atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -269,35 +345,44 @@ private static SerializableTransactionManager create(
         return transactionManager;
     }
 
-    private static SweepBatchConfig getSweepBatchConfig(AtlasDbConfig config) {
+    private static void validateRuntimeConfig(@Nullable AtlasDbRuntimeConfig runtimeConfig) {
+        if (runtimeConfig == null) {
+            log.warn("This service uses live reload configs. Please use the 'atlas-runtime' block for specifying"
```

Comment:
can we rephrase the first sentence to "AtlasDB now supports live reloadable configs."